src/app/technical-record/adr/adr-details/edit/__snapshots__/adr-details-edit.component.spec.ts.snap

```diff
@@ -337,7 +337,7 @@ exports[`AdrDetailsEditComponent should create with initialized form controls 1`
                 class="govuk-label govuk-!-font-weight-bold"
                 for="adrTypeApprovalNo"
               >
-                 ADR type approval number 
+                 ADR type approval number(optional) 
```

Comment:
should we have a space between the parenthesis and label?

Reply:
yes I agree. @rspasici  it should be like `olu (optional)` . Please lets update all where `optional` is used